pkg/maps/timestamp/timestamp_test.go

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package timestamp
+
+import (
+	"testing"
+
+	. "github.com/cilium/checkmate"
+
+	"github.com/cilium/cilium/api/v1/models"
+)
+
+// Hook up gocheck into the "go test" runner.
+func Test(t *testing.T) {
+	TestingT(t)
+}
+
+type TimestampSuite struct{}
+
+var _ = Suite(&TimestampSuite{})
+
+func (s *TimestampSuite) TestConvert(c *C) {
+	clockSource := &models.ClockSource{Mode: models.ClockSourceModeKtime}
```

Comment:
We are only testing happy path here. What if it isn't so?

Reply:
Perhaps we could leverage table testing here would clarifying the various use cases and DRY up this test?